reference/release-notes/1.26.0.md

```diff
@@ -0,0 +1,116 @@
+
+
+## CVEs
+
+The Anbox Cloud 1.26.0 release includes fixes from the respective upstreams and components for the following CVEs:
+
+| CVE | Affected Components |
+|-----|---------------------|
+| [CVE-2025-22869](https://nvd.nist.gov/vuln/detail/CVE-2025-22869) | AMS |
+| [CVE-2025-30215](https://nvd.nist.gov/vuln/detail/CVE-2025-30215) | NATS |
+| [CVE-2025-22872](https://nvd.nist.gov/vuln/detail/CVE-2025-22872) | Anbox |
+| [CVE-2025-43864](https://nvd.nist.gov/vuln/detail/CVE-2025-43864) | AMS |
+| [CVE-2025-43865](https://nvd.nist.gov/vuln/detail/CVE-2025-43865) | AMS |
+| [CVE-2023-21342](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 13 |
+| [CVE-2024-34739](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 13, 14 |
+| [CVE-2025-0077](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 15 |
+| [CVE-2025-0087](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 13, 14, 15 |
+| [CVE-2025-22425](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 13, 14 |
+| [CVE-2025-26422](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 15 |
+| [CVE-2025-26426](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 13, 14, 15 |
+| [CVE-2025-26427](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 13, 14 |
+| [CVE-2025-26428](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 13, 14, 15 |
+| [CVE-2025-26436](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 13, 14, 15 |
+| [CVE-2025-26440](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 14 |
+| [CVE-2025-26444](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 13, 14 |
+| [CVE-2025-26424](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 15 |
+| [CVE-2025-26442](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 13, 14, 15 |
+| [CVE-2025-26429](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 13, 14, 15 |
+| [CVE-2025-27363](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 13, 14 |
+| [CVE-2025-26420](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 13, 14 |
+| [CVE-2025-26421](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 13, 14, 15 |
+| [CVE-2025-26423](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 13, 14, 15 |
+| [CVE-2025-26425](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 14, 15 |
+| [CVE-2025-26430](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 15 |
+| [CVE-2025-26435](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 15 |
+| [CVE-2025-26438](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 13, 14, 15 |
+| [CVE-2023-35657](https://source.android.com/docs/security/bulletin/2025-05-01) |Android 13, 14, 15 |
```

Comment:
I am not quite sure what the policy is for this, for the VM lxd images, due to the following CVEs are landed into the Jammy kernel
```
    - https://ubuntu.com/security/CVE-2024-56658 5.15.0-136.147
    - https://ubuntu.com/security/CVE-2024-35864 5.15.0-136.147
    - https://ubuntu.com/security/CVE-2024-26928 5.15.0-136.147
    - https://ubuntu.com/security/CVE-2023-52664 5.15.0-139.149
```

Maybe we should mention those here.